### ISSUE TITLE
Incorrect FontMetrics on Windows via PrinterJob Graphics

### ISSUE THREAD

**Issue by teamacs** [AdoptOpenJDK/openjdk-build#443]
On Windows, the FontMetrics returned from Graphics.getFontMetrics(Font) within a PrinterJob are incorrect if the PageFormat has had the orientation explicitly set to PageFormat.LANDSCAPE or PageFormat.REVERSE_LANDSCAPE.  The issue is not present when explicitly setting the orientation to PageFormat.PORTRAIT. The attached code demonstrates the problem.

[JPSFontMetrics.zip](https://github.com/AdoptOpenJDK/openjdk-build/files/2287290/JPSFontMetrics.zip)

**Comment by karianna** [comment]
@teamacs Which binary is this with?

**Comment by karianna** [comment]
@teamacs which binary is this with?

**Comment by teamacs** [comment]
- OpenJDK 8 with Hotspot (Latest release)
- OpenJDK 8 with Eclipse OpenJ9 (Latest release)
- OpenJDK 10 with Hotspot (Most recent nightly build)
- OpenJDK 10 with Eclipse OpenJ9 (Most recent nightly build)

**Comment by karianna** [comment]
@teamacs Have you tried an Oracle JDK (8 or 10)?  I'm suspecting this is an OpenjDK upstream bug.

**Comment by teamacs** [comment]
Yes, we have tried and confirmed it returns correct FontMetrics using the following:
- Oracle JDK 8 (Windows and Mac OS)
- Oracle JDK 10 (Windows and Mac OS)
- OpenJDK 8 with Hotspot (Latest release on Mac OS)
- OpenJDK 10 with Hotspot (Latest release on Mac OS)

**Comment by karianna** [comment]
OK this is going to take some digging.  Adopt doesn't apply any source patches over and above OpenJ9 at Eclipse or OpenJDK/Hotspot at OpenJDK upstream so I can only imagine its a bundle lib issue.

**Comment by teamacs** [comment]
We have tested the following new releases and confirmed the issue also exists on Windows in these additional binaries:
- OpenJDK 11 with Hotspot (Latest release)
- OpenJDK 11 with Eclipse OpenJ9 (Latest release)
- Oracle JDK 11 (Latest release)

**Comment by mrbray99** [comment]
I have just come across this issue in an App that has been upgraded to OpenJDK 11.  Is there a known workaround?

**Comment by M-Davies** [comment]
@karianna This seems like an support issue rather than build. Could this be transferred over to https://github.com/AdoptOpenJDK/openjdk-support?
